activerecord/test/cases/adapters/postgresql/comment_test.rb

```diff
@@ -0,0 +1,87 @@
+require 'cases/helper'
+require 'support/schema_dumping_helper'
+
+class PostgresqlCommentTest < ActiveRecord::PostgreSQLTestCase
```

Comment:
Uh, seems like I was really sleepy this morning. Sorry for wasteing your time. Fixed!